python/ray/_raylet.pyx

```diff
@@ -17,6 +17,7 @@ import os
 import pickle
 import sys
 import _thread
+import ray
```

Comment:
Hmmm this seems a little suspicious/potentially problematic. Why is it needed?

Reply:
I can't really explain since I'm not sure myself but the inspiration is from https://github.com/ray-project/ray/pull/7031

Reply:
hmm yeah i'm just wondering if this could lead to some sort of circular import...

@simon-mo could you take a quick look at this?

Reply:
But it shouldn't be necessary? _raylet is imported after thirdparty_files are added to the path already:
https://github.com/ray-project/ray/blob/86b07410260fd42302c77d40209b0b80aa931590/python/ray/__init__.py#L40-L64

Reply:
Ah I see, then yeah I don't know why the macos wheels test is failing: https://travis-ci.com/github/ray-project/ray/jobs/457112601

Reply:
Mac OS wheels pass!
The solution was to use setproctitle wheels from pypi since in the latest version, they are already built. This means we don't have to build setproctitle from source ourselves. (It seems that windows doesn't like this tho, as windows is still trying to build the wheel, but I have an idea on how to fix this maybe)